### ISSUE TITLE
Cannot read any audio file anymore

### ISSUE THREAD

**Issue by liofeu** [clementine-player/Clementine#6937]
### Before posting

Please follow the steps below and check the boxes with [x] once you did the step.

- [x ] I checked the issue tracker for similar issues
- [ x] I checked the [changelog](https://github.com/clementine-player/Clementine/blob/master/Changelog) if the issue is already resolved
- [x ] I tried the latest Clementine build from [here](https://builds.clementine-player.org/)

### System information

Please provide information about your system and the version of Clementine used.

- Operating System: macOS Catalina 10.15.7
- Clementine version: last version 01 March 2021 (In the menu "Clementine > About Clementine" is grey, I cannot click on it..)

### Expected behaviour / actual behaviour

I cannot read any file from my library, even after doing a full rescan of the library. It behaves like if each audio file were 0 second and does it for all the files of the playlist. After having tried all the playlist files, Clementine returns the message "Playlist finished". 

### Steps to reproduce the problem (only for bugs)

Create a new playlist and import a sound file. You should be able to see it (title, duration, ...). Click play.

**Comment by liofeu** [comment]
It started suddenly today (I had a previous version of Clementine 1.4) while it was working with the same version a couple of days before. Then I downloaded the last Clementine version 1.4.0rc1-482-g0c099ab6f, and it is the same.

Same after restarting the computer.

**Comment by jbroadus** [comment]
So, clementine can still discover the music during scan and read tags? What kind of music files are they?

**Comment by liofeu** [comment]
Yes, it can still discover the music (at least it can open a new file 
and put it in a playlist)

The music files are my whole music library (mp3, oggs, mp4, ...) which I 
have used with Clementine for years

**Comment by liofeu** [comment]
Hum, I have just remembered that before this issue started, I unticked the option  "Music Library > Monitor the library". Then I ticked it back and I am now able to read my library

The weird thing is that I don't manage to reproduce the issue by unticking back the option, even by restarting Clementine...

**Comment by liofeu** [comment]
I reopen this issue because it comes back from time to time. Restarting Clementine sometimes solves the issue.

**Comment by nugged** [comment]
found very similar behavior - I was on very old clementine, installed latest 496 build, and any file I add – hanged on "reading tracks: 0%",

I tried to downgrade, one-by-one, jumped over few releases, so what I found:

when I downgraded at last to build 315, it started to work. And when now I upgraded back even to hours-ago fresh build 498 – it still works, **and adds files now.** Hurray!

Seems some database conversion issues didn't happen properly from very old to the very new version, but when I rolled back to 315 it was happened and afterward "all fixed" for upcoming updates. This is my assumption only.

**Comment by liofeu** [comment]
I confirm that downgrading to build 315, then upgrading back to 498 makes the issue move away... at least for now!

However, I was not running an old version before this issue arose, as @nugged  was. 

THank you very much for your comment @nugged. I moved to another player, but I'm back to clementine now.

**Comment by nugged** [comment]
to describe more of my stubbornness "to make it work", because no other options I had:

I jumped not sequentially back but by bigger leaps (but mostly skipping "translation" builds) through some, and eventually I went far away (randomly) to 315. Just to say that 315 is maybe not (or it is) "that one" :).

Then, on 315, I found this issue moved away, then I started to upgrade back, more sequentially, didn't found that issue returned after dozen updates, so become brave "just update to latest one", and it kept working.

So for now it works also for you, @liofeu – that helped for you - I am happy twice!
Because I SUFFER on MacOS without normal "folders" player and full-featured radio and playlists as well and library and in-files ratings as well... like Clementine.

@liofeu: which player you used in between, btw?

**Comment by liofeu** [comment]
I tried Strawberry, which is a fork of Clementine. It seems to be less buggy (at least for me, I have plenty of bugs now with Clementine), but there are less features than in Clementine...

Now, I'm back to Clementine build 332 (because of another bug, maybe related?? #6950 which starts from build 340) and the issue #6937 described in the current page started again... which I can handle from time to time by a non-sense trick in the Preferences (see my first post).

The main reason why I stayed on Clementine while having so many bugs (#6950, #6937, #6938 and other not reported) is because I have rated my entire library with Clementine, and I have not managed to export the ratings into all the audio files for whatever reasons. I assume that I can copy-paste one of the config files from "Library > Application support > Clementine"  to "Library > Application Support > Strawberry" in order to export my Clementine ratings to Strawberry, but I keep having some hopes that the bugs will be solved on Clementine soon :-) ...

**Comment by nugged** [comment]
I didn't bump in such ones, for now, at least I have music and radios for now playing, that was first need :), I suffered without folder playing to sort fresh founding.

Another issue that prevents the player from exit https://github.com/clementine-player/Clementine/issues/6953 but this one easy avoidable – just disable or login...

**Comment by nugged** [comment]
@liofeu – can you save ratings and play counts INTO files tags with clementine? Then other players who respect this might use that... from tags? (just thinking aloud, ... )

**Comment by nugged** [comment]
@liofeu did @jbroadus'es fix (#6951) removed one more issue for you :) ?

**Comment by liofeu** [comment]
> @liofeu – can you save ratings and play counts INTO files tags with clementine? Then other players who respect this might use that... from tags? (just thinking aloud, ... )

Now that Ive figured out that I can save statistics by using older builds or by waiting for the next release  build with  jbroaddus's fixes, I will do that soon!
Also, I have been quite careful since stat synchronization destroyed my whole ogg-file library some years ago (#4584), but this is all in the past :-)
